CHANGELOG.md

```diff
@@ -0,0 +1,9 @@
+# Sensu Go cookbook changelog
+
+This file is used to track changes made in recent versions of the Sensu Go
+cookbook. 
+
+## [Unreleased]
```

Comment:
We should still follow our changelog conventions with the various headers from plugins. Unless there is a really good reason not to.

Reply:
Nope, just me copying from the wrong place! :(

Reply:
what I was getting at was the missing `### Added` header.